cowbird/api/requests.py

```diff
@@ -69,7 +69,7 @@ def get_multiformat_body_raw(request, key, default=None):
 
 def get_multiformat_body(request, key, default=None, check_type=str, pattern=ax.PARAM_REGEX,
                          http_error=None, msg_on_fail=None):
-    # type: (Request, str, Any, Any, Optional[Union[str, bool]], Optional[Type[HTTPError]], Optional[str]) -> str
+    # type: (Request, str, Any, Any, Optional[Union[str, bool]], Optional[Type[HTTPError]], Optional[str]) -> Any
```

Comment:
Not sure about this, I used `Any` since `resource_id` is an `int`, and I didn't want to put restrictions on the type that could be used in the webhooks, if we ever want to use other types (`List`, `Tuple`, `bool`, etc.)

I am not sure if the `Type` keyword is right here. Wouldn't it expect an actual type such as `int` or `str` instead of an actual value in the returning value? And doesn't `Any` cover the `Tuple` case already?

I thought `Any` was good here since we can use any type. It just hadn't been tested yet with the `resource_id` and had just been using `str` values before.

Reply:
Normally, the `JSON` type should include all of those possible variations already.

Indeed, the `check_type` expects a type, not a value, hence the `Type[...]` typing. 
If you apply `Any`, there is no problem passing any value, which is wrong. 
Technically, `Any` does include `Type` and `Tuple` already, but it somewhat makes the type definition not useful at all.
The verbose `Union[Type[Any], Tuple[Type[Any], ...]]` explicitly tells that "any type" (not just "anything") is expected.

Reply:
I see the confusion, I thought you were talking in both comments about the returning type. I didn't realized the 2nd comment was about the `check_type`, my bad. :/
It makes sense now, and I changed it accordingly.

I also updated the returning type to just `JSON` (and not `Union[str, JSON]`), since `JSON` includes the `str` type already.